dotCMS/src/main/java/com/dotcms/storage/FileMetadataAPIImpl.java

```diff
@@ -654,7 +653,7 @@ public void putCustomMetadataAttributes(final String tempResourceId,
                         .storageKey(
                                 new StorageKey.Builder().group(metadataBucketName)
                                         .path(tempResourcePath(tempResourceId))
-                                        .storage(storageType).build())
+                                        .storage(StorageType.FILE_SYSTEM).build())
```

Comment:
why this?

Reply:
Currently, we don't have a way to clean up all the temporary stuff that would be created if the storage of choice was the db. But for temp files we do. So for now I think it makes sense if we use the file system to store temp files while we figure out a cleanup mechanism.